golem/network/concent/client.py

```diff
@@ -402,6 +402,7 @@ def _enqueue(self, key, msg):
         self._queue.put(msg)
 
     def income_listener(self, event, **kwargs):
+        logger.debug("income listener event: %s", event)
```

Comment:
https://github.com/golemfactory/golem-internal/wiki/Logging-guidelines#log-message-structure
```suggestion
         logger.debug("income listener. event=%r", event)
```